validation/pipeline/Jenkinsfile.destroy.airgap.rke2

```diff
@@ -0,0 +1,172 @@
+#!groovy
+// Load qa-jenkins-library dynamically based on parameter
+def libraryBranch = env.QA_JENKINS_LIBRARY_BRANCH ?: 'main'
+library "qa-jenkins-library@${libraryBranch}"
+
+node {
+    def testsPath = "./tests"
+    def qaInfraPath = "./qa-infra-automation"
+    def tofuModulePath = "${qaInfraPath}/tofu/aws/modules/airgap"
+    def testsBranch = env.RANCHER_TEST_REPO_BRANCH ?: "main"
+    def testsRepo = env.RANCHER_TEST_REPO_URL ?: "https://github.com/rancher/tests"
+    def qaInfraBranch = env.QA_INFRA_REPO_BRANCH ?: "main"
+    def qaInfraRepo = env.QA_INFRA_REPO_URL ?: "https://github.com/rancher/qa-infra-automation"
+    def targetWorkspace = env.TARGET_WORKSPACE
+    
+    wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm']) {
+        withFolderProperties {
+            paramsMap = []
+            params.each {
+                if (it.value != null && it.value.toString().trim() != "") {
+                    paramsMap << "$it.key=$it.value"
+                }
+            }
+            
+            withCredentials([
+                string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'AWS_ACCESS_KEY_ID'),
+                string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 'AWS_SECRET_ACCESS_KEY')
```

Comment:
I'm sure you checked but since I'm new to looking at stuff using the shared library, are these not available somewhere in there?